2021/11/main.ts

```diff
@@ -4,4 +4,4 @@ import { run as partTwo } from "./part-two.ts";
 
 const data = await fetchData("./input.txt");
 console.log("Part 1:", partOne(data)); // 1585
-console.log("Part 2:", partTwo(data));
+console.log("Part 2:", partTwo(data)); // 282
```

Comment:
Correct answer was 382 by using this code: https://github.com/sk1talets/advent-of-code/blob/main/2021/11/script.2.js